apps/monitoring/backend/monitoring/settings.py

```diff
@@ -39,4 +39,5 @@ class Config:
     extensions=[],
     webhooks=[],
     author="Saleor Commerce"
+    # TODO Add brand.logo.default
```

Comment:
@przlada FYI. We dont have to add it now. I added file to frontend's /public dir. It can be referenced with `https://monitoring.saleor.app/logo.png`